tools/gfastats/gfastats.xml

```diff
@@ -0,0 +1,298 @@
+                            <option value="gfa.gz">GFA.gz</option>
+                    </param>
+                    <when value="fasta">
+                        <expand macro="length_macro"/>
+                    </when>
+                    <when value="fasta.gz">
+                        <expand macro="length_macro"/>
+                    </when>
+                    <when value="fastq"/>
+                    <when value="fastq.gz"/>
+                    <when value="gfa"/>
+                    <when value="gfa.gz"/>
+                </conditional>
+            </when>
+            <when value="statistics">
+                <conditional name="statistics_condition">
+                    <param name="selector" type="select" label="Report mode">
+                        <option value="size">Scaffold, contig or gap sizes (--out-size)</option>
+                        <option value="coordinates">AGP, contig or gap coordinates (--out-coord)</option>
+                        <option value="assembly">Genome assembly statistics (--nstar-report)</option>
+                        <option value="sequence">Sequence statistics (--seq-report)</option>
+                    </param>
+                    <when value="size">
+                        <param argument="--out-size" type="select" label="Feature for reporting sizes"
+                            help="Generate a tabular file with the sequence sizes">
+                            <option value="s">Scaffolds</option>
+                            <option value="c">Contigs</option>
+                            <option value="g">Gaps</option>
+                        </param>
+                    </when>
+                    <when value="coordinates">
+                        <param argument="--out-coord" type="select" label="BED coordinares feature"
+                            help="Generates bed coordinates of given feature. Default: agp">
+                            <option value="a">AGP</option>
+                            <option value="c">Contigs</option>
+                            <option value="g">Gaps</option>
+                        </param>
+                    </when>
+                    <when value="assembly">
+                        <param name="expected_genomesize" type="integer" min="0" optional="true"
+                            label="Expected genome size" help="Estimated genome size. This parameter is optional, but required for NG* statistics."/>
+                    </when>
+                    <when value="sequence">
+                        <param argument="--out-sequence" type="boolean" truevalue="--out-sequence" falsevalue="" checked="false"
+                            label="Report actual sequence" help="It reports also the actual sequence"/>
+                    </when>
+                </conditional>
+            </when>
+        </conditional>
+        <param argument="--tabular" type="boolean" truevalue="--tabular" falsevalue="" checked="false"
```

Comment:
where does this param play a role in the outputs?

Reply:
Yes, indeed by default the output format should be text; I'll set it as default.